invadedhouse/run.py

```diff
@@ -16,6 +19,10 @@
 
 
 def main():
+    cprint('   ____                       __         __  __ __                   ', 'cyan')
+    cprint('  /  _/___   _  __ ___  _ ___/ /___  ___/ / / // /___   __ __ ___  ___', 'cyan')
+    cprint(' _/ / / _ \ | |/ // _  `// _  // -_)/ _  / / _  // _ \ / // /(_-</  -_)', 'cyan')
+    cprint('/___//_//_/ |___/ \_,_/ \__,_/ __/ \_,_/  /_//_/ ___/ \__,_//___/ \__/ ', 'cyan')
```

Comment:
very cool addition!!

don't forget to add another print here so there is an extra line when the game starts up